### PR TITLE
Feature: input components

### PR DESCRIPTION
**Pull request by mibsbalsante**

### CODE REVIEW

File: components/atoms/input/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  type: PropTypes.string,
+  checked: PropTypes.bool,
+  disabled: PropTypes.bool,
+  id: PropTypes.string,
+  name: PropTypes.string,
+  placeholder: PropTypes.string,
+  multiline: PropTypes.bool,
+  required: PropTypes.bool,
+  value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  handleChange: PropTypes.func.isRequired,
```
Comment (dleitee) [inline components/atoms/input/index.js]: I think that `handleChange` should be optional and the default should be `() => {}`

File: components/molecules/checkbox/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <label className={styles.checkbox} role="checkbox" aria-checked={isChecked}>
+      <Input {...props} type="checkbox" className={inputClasses} checked={isChecked} />
+    </label>
+  )
+}
+
+Checkbox.propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string.isRequired,
```
Comment (dleitee) [inline components/molecules/checkbox/index.js]: none of it should be required.

File: components/molecules/radio/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <label className={styles.radio} role="radio" aria-checked={isChecked}>
+      <Input {...props} type="radio" className={inputClasses} checked={isChecked} />
+    </label>
+  )
+}
+
+Radio.propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string.isRequired,
```
Comment (dleitee) [inline components/molecules/radio/index.js]: none of it should be required.

File: src/images/check.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="7" height="5" viewBox="0 0 7 5">
```
Comment (dleitee) [inline src/images/check.svg]: move it to `components/assets/...`

**Review by dleitee — CHANGES_REQUESTED** [review]

**Comment by mibsbalsante** [comment]
@dleitee can you check if is everything ok now, please? 😄

File: components/molecules/radio/index.stories.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types'
+import React from 'react'
```
Comment (beatriz1304) [inline components/molecules/radio/index.stories.js]: Import Component here like
`import React, {Component} from 'react'`

Comment (beatriz1304) [reply components/molecules/radio/index.stories.js]: You are using Fragment too 
`import React, {Component, Fragment} from 'react'`

File: components/molecules/radio/index.stories.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+          handleChange: this.handleChange,
+        }
+
+        return React.cloneElement(child, childProps)
+      })}
+    </React.Fragment>
+  )
+}
+
+CustomRadioGroup.propTypes = {
```
Comment (beatriz1304) [inline components/molecules/radio/index.stories.js]: Put prop validation inside the class as a `static property`
`static propTypes = {}`

Comment (beatriz1304) [reply components/molecules/radio/index.stories.js]: Do that with the default Props too

File: src/config/css-variables.js [hunk 7]
```diff
@@ -3,9 +3,19 @@ const styles = {}
 styles['font-default'] = "'Source Sans Pro', sans-serif"
 styles['font-header'] = "'Poppins', sans-serif"
 
+styles['color-text'] = '#000'
```
Comment (beatriz1304) [inline src/config/css-variables.js:6]: We need to define the color pallette, remember that we've talked about the color variable names like `color-primary, color-primary-dark ` and so on

File: components/molecules/checkbox/index.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  const { className, isChecked, disabled } = props
+  const inputClasses = classNames({
+    [className]: true,
+    [styles.input]: true,
+    [styles.checked]: isChecked,
+    [styles.disabled]: disabled,
+  })
+
+  return (
+    <label className={styles.checkbox} role="checkbox" aria-checked={isChecked}>
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.js]: I think this label could be a separated component that will receive a children

File: components/molecules/checkbox/index.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+      <Input {...props} type="checkbox" className={inputClasses} checked={isChecked} />
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.js]: There was a PR mine that DLeite told me to remove the `...props` and write down all the properties to be clear for the user what he really needs to send

File: components/molecules/checkbox/index.js [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    </label>
+  )
+}
+
+Checkbox.propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string.isRequired,
+  isChecked: PropTypes.bool.isRequired,
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.js]: In this line you put that property is required and on the `defaultProps` you attributed a value for this.

File: components/molecules/checkbox/index.stories.js [hunk 11]
```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { withKnobs, number } from '@storybook/addon-knobs/react'
+
+import Checkbox from './index'
+
+class CustomCheckboxGroup extends React.Component {
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.stories.js]: Import `Components`

File: components/molecules/checkbox/index.stories.js [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+  state = { values: [1] }
+
+  includesValue = val => this.state.values.includes(val)
+
+  handleChange = e => {
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.stories.js]: Change the parameter name to be clear from `e` to `event` or other thing

File: components/molecules/checkbox/index.stories.js [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+
+    const newValues = this.includesValue(value)
+      ? values.filter(arrVal => arrVal !== value)
+      : [...values, value].sort()
+
+    this.setState({ values: newValues })
+  }
+
+  render = () => (
+    <React.Fragment>
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.stories.js]: Import `Fragment`

File: components/molecules/checkbox/index.stories.js [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+      <h1>Selected: {this.state.values.join(', ')}</h1>
+
+      {React.Children.map(this.props.children, child => {
+        const childProps = {
+          key: child.props.value,
+          name: 'checkbox[]',
```
Comment (beatriz1304) [inline components/molecules/checkbox/index.stories.js]: All the child will have the same name? Is it correctly?

File: components/molecules/radio/styles.css [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+    height: 5px;
+    background: white;
+    opacity: 0;
+  }
+
+  &:focus {
+    border-color: var(--color-input-focus);
+
+    &::before {
+      box-shadow: 0 0px 1px 1px #91d0d9;
```
Comment (beatriz1304) [inline components/molecules/radio/styles.css]: Use this color as a var

File: components/molecules/radio/styles.css [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+    height: 18px;
+    background: var(--color-blue);
+    border: 1px solid var(--color-blue);
+    outline: 0;
+  }
+
+  &::after {
+    width: 5px;
+    height: 5px;
+    background: white;
```
Comment (beatriz1304) [inline components/molecules/radio/styles.css]: Use that color as a var

File: components/molecules/checkbox/index.stories.js [hunk 17]
```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types'
+import React, { cloneElement, Children, Component, Fragment } from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { withKnobs, number } from '@storybook/addon-knobs/react'
+
+import Checkbox from './index'
+
+class CustomCheckboxGroup extends Component {
```
Comment (dleitee) [inline components/molecules/checkbox/index.stories.js:9]: We can move it to components later, what do you think?

**Review by dleitee — DISMISSED** [review]
Awesome!!!!

**Review by beatriz1304 — APPROVED** [review]

File: components/molecules/checkbox/index.stories.js (hunk 17 above)
Comment (mibsbalsante) [reply components/molecules/checkbox/index.stories.js:9]: We can try it in the future 🎉

**Pull request merged**